File: DB Project/divms/examples/sample_mysql.php
<?php
echo("<html><head><title>Sample PHP Script</title></head>\n");

//  hostname, username, password
$dbcnx = mysql_connect("dbdev.cs.uiowa.edu:3306","dbdevtest","XiuyD=jP");
if (!$dbcnx) {
  echo( "<P>Unable to connect to the " .
        "database server at this time.</P>\n" );
  exit();
}
mysql_select_db("WorksOn",$dbcnx);
if (! @mysql_select_db("WorksOn") ) {
  echo( "<P>Unable to locate the sample " .
        "database at this time.</P>" );
  exit();
}
# do a simple query and print it out
$sql_query="SELECT ename, salary, dno FROM Emp";

$result_set = mysql_query($sql_query);
if (!$result_set) {
  echo("<P>Error performing query: " .
       mysql_error() . "</P>");
  exit();
}

echo("<h2>" . $sql_query . "</h2>");
echo("<table><tr><th>Name</th><th>Salary</th><th>dno</th></tr>\n");
while ( $row = mysql_fetch_array($result_set) ) {
  echo("<tr><td>" . $row["ename"] . "</td>\n" .
       "     <td>" . $row["salary"] . "</td>\n" .
       "     <td>" . $row["dno"] . "</td>\n" .
       "</tr>\n");
}
echo ("</table>\n");
echo ("</body></html>");
?>

File: DB Project/divms/examples/sample_mysql.py
import MySQLdb

s = """ %s """

def index(req):
   message = "<html><head><title>Sample Python Script</title></head>\n"
   # connect
   dbcnx = MySQLdb.connect(host="dbdev.cs.uiowa.edu",port=3306,user="dbdevtest",passwd="XiuyD=jP",db="WorksOn")
   
   sqlquery="""SELECT ename, salary, dno FROM Emp;"""
   
   # create a database cursor
   cursor = dbcnx.cursor()
   
   # execute SQL select 
   cursor.execute(sqlquery)
   
   message += "<h2> %s </h2>"
   message %= sqlquery
   message += "<table><tr><th>Name</th><th>Salary</th><th>dno</th></tr>\n"
   
   # get the number of rows in the resultset
   numrows = int(cursor.rowcount)
   
   # get and display one row at a time
   for x in range(0,numrows):
       row = cursor.fetchone()
       message += "<tr><td> %s </td>\n"
       message %= row[0]
       message += "    <td> %s </td>\n"
       message %= row[1]
       message += "    <td> %s </td></tr>\n"
       message %= row[2]
   
   message += "</table>\n"
   message += "</body></html>\n"
   cursor.close ()
   dbcnx.close ()
   return s % (message)
   
       
